test.sh

```diff
@@ -0,0 +1,30 @@
+docker-compose up -d
+echo Please wait...
+docker container exec -i db-container mysql bbdb < baseball.sql -ppass

```

Comment:
I used Source here to upload SQL database... and it took me like 10-15 minutes to run this section lol. Was this method faster for you here??

Reply:
I found copying into the container and then using mysqld command to be less than 5 min.